elvis.config

```diff
@@ -3,42 +3,41 @@
    elvis,
    [
     {config,
-     [#{dirs => ["src", "test"],
+     [#{dirs => ["src"],
         filter => "*.erl",
-        rules => [{elvis_style, line_length, #{limit => 80}},
```

Comment:
I just git clone apns run 
```erlang
make elvis
```
it just update elvis.config all by this 

Maybe I should revert elvis.config?